pkg/cli/util/args.go

```diff
@@ -14,6 +18,8 @@ var (
 	VClusterNameOnlyUseLine string
 
 	VClusterNameOnlyValidator cobra.PositionalArgs
+
+	ErrNonInteractive = errors.New("terminal is not interactive")
```

Comment:
I think this error is a little awkward when you forget to put an argument it will yield `terminal is not interactive` which is not really what you would want. It should say something like `please specify vcluster ... [VCLUSTER_NAME]` or something more understandable

Reply:
Nvm see you do the check in the caller

Reply:
The way I'm calling it, it will report the same error it did before for none interactive usage since each of the three places its called it checks for `ErrNonInteractive` and then falls back to trying the validation that was there before.  I thought about adding a `cobra.PositionalArgs` to the arguments, so it could take care of that automatically, but thought it might be seen as overkill.  So like [this](https://github.com/zerbitx/vcluster/pull/1/files?w=1)